cypress/integration/data-import/import-by-oclc.spec.js

```diff
@@ -0,0 +1,50 @@
+import TopMenu from '../../support/fragments/topMenu';
+import InventoryActions from '../../support/fragments/inventory/inventoryActions';
+import testTypes from '../../support/dictionary/testTypes';
+import permissions from '../../support/dictionary/permissions';
+import Users from '../../support/fragments/users/users';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+import InventoryViewSource from '../../support/fragments/inventory/inventoryViewSource';
+import InventorySearch from '../../support/fragments/inventory/inventorySearch';
+
+let user;
+const oclc = '1007797324';
+
+describe('data-import: import by OCLC', () => {
+  before('create user', () => {
+    cy.createTempUser([
+      permissions.uiQuickMarcQuickMarcBibliographicEditorAll.gui,
+      permissions.inventoryAll.gui,
+      permissions.uiInventorySingleRecordImport.gui,
+    ])
+      .then(userProperties => {
+        user = userProperties;
+        cy.login(user.username, user.password);
+      });
+  });
+
+  beforeEach('navigate to inventory', () => {
+    cy.visit(TopMenu.inventoryPath);
+  });
+
+  after('delete test data', () => {
+    cy.getInstance({ limit: 1, expandAll: true, query: `"oclc"=="${oclc}"` })
+      .then(instance => {
+        InventoryInstance.deleteInstanceViaApi(instance.id);
+      });
+    Users.deleteViaApi(user.userId);
+  });
+
+  it('C193953 Overlay existing Source = MARC Instance by import of single MARC Bib record from OCLC (Prokopovych)', { tags: [testTypes.smoke] }, () => {
+    InventoryActions.import(oclc);
+    InventoryInstance.viewSource();
+    InventoryViewSource.contains('999\tf f\t‡s');
```

Comment:
Can you please explain the difference between the test case and the test: Verify that 999 ff $s and $i remain the same  and 999\tf f\t‡s ?  Do we need to update the test case?

Reply:
In first case I click view source after import
In second case I click view source after inventory search
And verify need here to be sure that right source was opened

Reply:
In test I verified 999 f f but with tabs \t as in test case
I assume you mean difference between $i and $s - it depends of oclc code. In test case author suggest to pick code from third-party site. May be author write test case for one book but I picked another